### PR TITLE
Introduced DebouncingClickListener

### LINKED ISSUE #628: [Help wanted] Speaker icon quickly tapped issue
## Overview (Required)
- when I tapped speaker icon quickly, SpeakerDetailActivity launch twice
- press back button twice then speaker icon is not shown
- the icon will be shown by re-bind with scroll or move to other Fragment

ezgif-4-92c871ff15

### PR DESCRIPTION
**Pull request by oldergod**
## Issue
- close #628 

## Overview (Required)
- Taken from ButterKnife, this class helps dealing with multiple OnClick calls when the main thread hangs. See: https://github.com/JakeWharton/butterknife/blob/e78507711fe8a7c637ee61c44a7b09f1be8ff9f6/butterknife/src/main/java/butterknife/internal/DebouncingOnClickListener.java

### CODE REVIEW

File: app/src/main/java/io/github/droidkaigi/confsched2018/presentation/common/view/SpeakersAdapter.kt [hunk 1]
```diff
@@ -7,13 +7,11 @@ import android.view.View
 import android.view.ViewGroup
 import io.github.droidkaigi.confsched2018.databinding.ItemSpeakersSummaryLayoutBinding
 import io.github.droidkaigi.confsched2018.model.Speaker
+import io.github.droidkaigi.confsched2018.util.DebouncingOnClickListener
 
-/**
- * Created by furusin on 2018/01/17.
- */
```
Comment (takahirom) [inline app/src/main/java/io/github/droidkaigi/confsched2018/presentation/common/view/SpeakersAdapter.kt]: Thanks 🙇

**Comment by takahirom** [comment]
I don't know about this 👍

**Comment by takahirom** [comment]
Can you add to README? 👀 
https://github.com/DroidKaigi/conference-app-2018#credit

**Comment by oldergod** [comment]
Done, thank you.

**Comment by takahirom** [comment]
LGTM 👍 I learned by this.

**Pull request merged**